app/controllers/groups_controller.rb

```diff
@@ -173,7 +171,24 @@ def invite
 
   def accept_invite
     @invite = UserInvite.where(token: params[:token]).first
-    @added = @invite.accept_invite(cmr_client, current_user.urs_uid, token)
+
+    urs_search_response = cmr_client.search_urs_users(@invite.user_email)
+
+    recipient_uid = urs_search_response.body.fetch('users', []).find { |user| user['email_address'] == @invite.user_email }['uid']
+
+    @added = recipient_uid && @invite.accept_invite(cmr_client, recipient_uid, token)
+  end
+
+  def urs_search
+    render json: render_users_from_urs(
+      search_urs(params[:search])
+    )
+  end
+
+  def provided_urs_users
```

Comment:
I looked for where this endpoint was being used but didn't find anything. Did I miss it or is this not being used?